### DISCUSSION TITLE
灌水，这个地方让我感觉回到了200x年的互联网

### DISCUSSION THREAD

**Discussion by 1451374815** [share121/inter-knot#1136] [其他]
没有小丑，没有巨婴，没有节奏，只有正常的讨论，比较慢的网速和图片加载，比较复古的页面，就非常像10年前的互联网

**Comment by likeseele** [comment]
你是gg还是mm
![Uploading GCK}BDF]QT9MRB97K_[Y{EL.png…]()
![Uploading GCK}BDF]QT9MRB97K_[Y{EL.png…]()

**Comment by Mu-7593** [comment]
我稀饭你

**Reply by 1451374815** [reply]
啊

**Comment by Dave-12138** [comment]
@likeseele  你看见有uploading就说明图还没上传完，要等上传完再点提交

**Comment by Kamisato100** [comment]
唯一不还原的地方可能是二次元头像过于潮流，少了几分复古土味
